lib/jxl/base/status.h

```diff
@@ -287,6 +185,25 @@ JXL_NORETURN inline JXL_NOINLINE bool Abort() {
     }                                                    \
   } while (0)
 
+#if JXL_IS_DEBUG_BUILD
+// Debug: fatal check.
+#define JXL_ENSURE(condition)                     \
+  do {                                            \
+    if (!(condition)) {                           \
+      ::jxl::Debug("JXL_ASSURE: %s", #condition); \
```

Comment:
missed one rename ;-)

Reply:
Ooops, was searching for `JXL_ASSURE(`...